### PR TITLE
Close-to-final Spanner fixes

### PR DESCRIPTION
**Pull request by jskeet**
(Review one commit at a time.)

### CODE REVIEW

File: apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/Google.Cloud.Spanner.Data.Tests.csproj [hunk 1]
```diff
@@ -32,7 +32,7 @@
   <ItemGroup>
     <ProjectReference Include="..\Google.Cloud.Spanner.Data.CommonTesting\Google.Cloud.Spanner.Data.CommonTesting.csproj" />
     <Compile Update="V1\SessionPoolTests.*.cs">
```
Comment (amanda-tarafa) [inline apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/Google.Cloud.Spanner.Data.Tests.csproj]: Just to double check that the remaining V1 here is needed.

**Comment by jskeet** [comment]
Urgh - I'd unloaded the test projects in the EF Core solution. Will amend the second commit.

**Comment by amanda-tarafa** [comment]
It's still broken though.

**Comment by jskeet** [comment]
Amended the second commit. Fingers crossed...

File: apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/Google.Cloud.Spanner.Data.Tests.csproj (hunk 1 above)
Comment (jskeet) [reply apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/Google.Cloud.Spanner.Data.Tests.csproj]: Yup. But the other one was broken. Presumably the "depends on" name is relative to the source file.

**Pull request merged**